avocado/utils/ssh.py

```diff
@@ -253,7 +253,10 @@ def copy_files(self, source, destination, recursive=False):
             cmd = path_utils.find_command('scp')
         except path_utils.CmdNotFoundError as exc:
             raise exc
-        options = self._dash_o_opts_to_str(self.DEFAULT_OPTIONS)
+        options = list(self.DEFAULT_OPTIONS)
+        if self.user is not None:  # Prevent failure from unspecificed user
```

Comment:
The spell checker (which runs on our CI) caught the following mispelling: "unspecificed", which should be "unspecified.

Reply:
I've fixed that and force-pushed the change.